src/adv.py

```diff
@@ -32,20 +40,63 @@
 room['narrow'].w_to = room['foyer']
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
+room['treasure'].d_to = room['stairway']
+room['stairway'].u_to = room['treasure']
+room['stairway'].n_to = room['secretroom']
+room['secretroom'].s_to = room['stairway']
+room['secretroom'].k_to = room['realtreasureroom']
+
+player = Player(room['outside'])
+
+def direction_successful(direction, current_room):
+    attribute = direction + "_to"
+
+    if hasattr(current_room, attribute):
+        return getattr(current_room, attribute)
+    elif(attribute == "q_to"):
+        return
+    else:
+        print("You can't go that way")
+        return current_room
+
+while True:
+    print(player.current_room.name)
+    print(player.current_room.description)
+    print(player.current_room.equipment)
+
+    player_input = input("\n>").lower().split()
```

Comment:
nice job splitting this into a list now, it will set you up for dealing with multi-line inputs later such as "take item"